stratum/hal/lib/common/common.proto

```diff
@@ -38,8 +38,8 @@ enum Platform {
   PLT_GENERIC_TOMAHAWK_PLUS = 4;
   PLT_MLNX_SN2700 = 9;
   PLT_P4_SOFT_SWITCH = 10;
-  PLT_BAREFOOT_TOFINO = 11;
-  PLT_BAREFOOT_TOFINO2 = 12;
```

Comment:
People will have an issue if they still use the old format. I would suggest using `[deprecated = true];` and add a new enum item.

Reply:
Makes sense, deprecate for the next release and remove in the next.

Reply:
right, and remember to let people (slack, TST, mailing list) know they need to change the config.
Not sure if the protobuf library will show any warning about using deprecated enum.